libafl/src/lib.rs

```diff
@@ -194,7 +194,6 @@ mod tests {
 
         let mut harness = |_buf: &[u8]| ExitKind::Ok;
         let mut executor = InProcessExecutor::new(
-            "main",
```

Comment:
Names for executors are no longer en-vogue?

Reply:
not needed, we can support multiple executors in another way